packages/api-client/src/client/ClientClassification.ts

```diff
@@ -22,4 +22,5 @@ export enum ClientClassification {
   LEGAL_HOLD = 'legalhold',
   PHONE = 'phone',
   TABLET = 'tablet',
+  UNKNOWN = '?',
```

Comment:
This is needed when we create a new client from a message we receive. 
This message only contains the id of the device but the type (desktop, mobile, tablet or legalhold) is unknown at this moment